pluginsdk/support/bundlepublisherutil/bputil.go

```diff
@@ -0,0 +1,257 @@
+// Package bundlepublisherutil provides helper functions for plugins
+// implementing the BundlePublisher interface.
+// BundlePublisher plugins should use this package as a way to have a
+// standarized name for bundle formats in their configuration, and avoid the
+// re-implementation of bundle parsing logic of formats supported in this
+// package.
+package bundlepublisherutil
```

Comment:
I don't know how many of these support packages we may have, but I imagine its possible that there may be helpers that are useful across multiple bundle types. Considering that, how would you feel about this package can just being isolated to bundle formatting? like `bundleformat`. It can have types like `bundleformat.Format` and methods like `bundleformat.Format` and `bundleformat.NewFormatter`, etc.

Reply:
Having this isolated to bundle formatting in a `bundleformat` package looks good to me.